docs/feature/cluster/index.md

```diff
@@ -0,0 +1,202 @@
+
+:::{rubric} Details
+:::
+
+Sharding and partitioning are techniques used to distribute data evenly across
+multiple nodes in a cluster, ensuring data scalability, availability, and
+performance.
+
+Replication can be applied to increase redundancy, which reduces the chance of
+data loss, and to improve read performance.
+
+:Sharding:
+
+    In CrateDB, tables are split into a configured number of shards. Then, the
+    shards are distributed across multiple nodes of the database cluster.
+    Each shard in CrateDB is stored in a dedicated Lucene index.
+
+    You can think of shards as a self-contained part of a table, that includes
+    both a subset of records and the corresponding indexing structures.
+
+    Figuring out how many shards to use for your tables requires you to think about
+    the type of data you are processing, the types of queries you are running, and
+    the type of hardware you are using.
+
+:Partitioning:
+
+    CrateDB also supports splitting up data across another dimension with
+    partitioning.
+    Tables can be partitioned by defining partition columns.
+    You can think of a partition as a set of shards.
+
+    - Partitioned tables optimize access efficiency when querying data, because only
+      a subset of data needs to be addressed and acquired.
+    - Each partition can be backed up and restored individually, for efficient operations.
+    - Tables allow to change the number of shards even after creation time for future
+      partitions. This feature enables you to start out with few shards per partition,
+      and scale up the number of shards for later partitions once traffic
+      and ingest rates increase over the lifetime of your application or system.
+
+:Replication:
+
+    You can configure CrateDB to replicate tables. When you configure replication,
+    CrateDB will ensure that every table shard has one or more copies available
+    at all times.
+
+    Replication can also improve read performance because any increase in the
+    number of shards distributed across a cluster also increases the
+    opportunities for CrateDB to parallelize query execution across multiple nodes.
+
+::::
+
+::::{grid-item}
+:class: rubric-slim
+:columns: auto 3 3 3
+
+:::{rubric} Concepts
+:::
+- {ref}`crate-reference:concept-clustering`
+- {ref}`crate-reference:concept-storage-consistency`
+- {ref}`crate-reference:concept-resiliency`
+
+:::{rubric} Reference Manual
+:::
+- {ref}`crate-reference:ddl-sharding`
+- {ref}`crate-reference:partitioned-tables`
+- {ref}`Partition columns <gloss-partition-column>`
+- {ref}`crate-reference:ddl-replication`
+
+{tags-primary}`Clustering`
+{tags-primary}`Sharding`
+{tags-primary}`Partitioning`
+{tags-primary}`Replication`
+::::
+
+:::::
+
+
+## Synopsis
+With a monthly throughput of 300 GB, partitioning your table by month,
+and using six shards, each shard will manage 50 GB of data, which is
+within the recommended size range (5 - 100 GB).
```

Comment:
Afaik the recommended upper limit of one shard is 50GB. There isn't really a lower limit, lower limits are more like recommendations to rather create a few shards with bigger sizes than a lot of smaller ones (less shards are better).

Reply:
Good to know. So, the 100 GB limit should be edited away, also on the other spot where I picked it up?

Reply:
Yes, curios were it is documented like this.

Reply:
Fixed with d71839741.

Reply:
ES also recommends the same:

> Experience shows that shards between 10GB and 50GB typically work well for many use cases, as long as the per-shard document count is kept below 200 million.
>
> -- https://www.elastic.co/guide/en/elasticsearch/reference/current/size-your-shards.html#shard-size-recommendation